### DISCUSSION TITLE
Dual shock 4 generic no connect to switch

### DISCUSSION THREAD

**Discussion by REXEL-OWO** [ndeadly/MissionControl#766] [Q&A]
Hi, I have an issue with a generic PS4 controller not connecting, the Switch when trying to connect it turns off the controller and doesn't detect it, the controller just stops pairing. I have Mission Control installed and it works with other generic Chinese controllers and Wii motes, Please help :c

**Comment by ndeadly** [comment]
Yeah this is "normal" unfortunately. I can't guarantee support for all clone controllers. There are a handful that seem to pair correctly but the console still disconnects them for some reason. It might be possible to fix it by patching the Bluetooth service, but I can't do so without a physical controller to debug with and find the issue.

**Reply by REXEL-OWO** [reply]
how i can help, i can send you a driver of the controller clone?

**Reply by ndeadly** [reply]
> a driver of the controller clone

Not sure what you mean here, probably a translation issue. What I would need is one of the actual physical controllers that isn't working, so I can debug the connection process myself.

**Reply by REXEL-OWO** [reply]
Do these drivers work? They are the ones related to the PS4 clone control by USB      https://drive.google.com/drive/folders/1IKF2swfP8fYnOgQ6lrHq4KGmpxf3Q1c_?usp=drive_link

**Reply by ndeadly** [reply]
I have no idea what this is you linked, I don't have access to the directory. But some random PC driver is not going to work with the Switch. The only thing that will help is a physical controller in my hands.

**Reply by REXEL-OWO** [reply]
oh ok well thank for everything
